packages/dds/tree/src/test/shared-tree/editing.spec.ts

```diff
@@ -3337,6 +3337,159 @@ describe("Editing", () => {
 			const child = tree.branch();
 			abortTransaction(child);
 		});
+
+		describe("No Change constraint", () => {
```

Comment:
There is a "Constraints" describe block higher in this file that these tests can be moved to.